_drafts/react-example-todomvc.md

```diff
@@ -0,0 +1,129 @@
+---
+title: TodoMVC example, a React version
+layout: default
+author: Stéphane Legrand
+---
+
+[TodoMVC](http://todomvc.com/) is a project which offers the same Todo application implemented using MV* concepts in most of the popular JavaScript MV* frameworks. One of the aim of this project is to give a possibility to make a fair comparison between several frameworks by coding the same application. A js_of_ocaml version using the React module is now available. Check out the [source code](https://github.com/slegrand45/examples_ocsigen/tree/master/jsoo/todomvc-react) and the [demo](http://slegrand45.github.io/examples_ocsigen.site/jsoo/todomvc-react/). Comments and pull requests are welcome!
```

Comment:
Wrapping at 80 characters (or even better, something like 72) would make the text much easier to edit/proof-read. Also: "aim**s** of this project", "... is to enable a fair comparison between ..."